storybook/qmlTests/tests/tst_SwapModal.qml

```diff
@@ -289,9 +289,7 @@ Item {
 
             // check default value of network comboBox, should be mainnet
             compare(root.swapFormData.selectedNetworkChainId, -1)
-            const networkComboBoxText = findChild(networkComboBox.control.contentItem, "contentItemText")
-            verify(!!networkComboBoxText)
-            compare(networkComboBoxText.text, root.swapAdaptor.filteredFlatNetworksModel.get(0).chainName)
+            compare(root.swapAdaptor.filteredFlatNetworksModel.get(0).chainId, 11155111 /*Sepolia Mainnet*/)
```

Comment:
why arent we comparing the text seen on the network selector anymore?

Reply:
there's no text in the Swap modal flows:

![image](https://github.com/status-im/status-desktop/assets/5377645/c320e252-fffa-4fa6-9902-38e39bebdc5b)

Reply:
then maybe we can test the presence of the relevant icon?

Reply:
I'm doing that below in the `for` loop but then... half of the networks have this icon `"network/Network=Testnet"` so it's not very useful :laughing: Any better idea?